### DISCUSSION TITLE
Mappings between Skywalking Agent and Skywalking APM versions

### DISCUSSION THREAD

**Discussion by peoplewhoworkhard** [apache/skywalking#9715] [Q&A] [answered]
HELLO, everyone!
	I would like to know the agent version is Apache-Skywalk-Java-Agent-8.9.0, which APM versions can be used?

**Comment by wu-sheng** [comment, marked as answer]
https://skywalking.apache.org/docs/main/next/en/setup/service-agent/agent-compatibility/

Read docs before asking questions.
